source/_components/sensor.azure_maps_travel_time.markdown

```diff
@@ -0,0 +1,145 @@
+ha_iot_class: "Cloud Polling"
+ha_release: 0.83
+---
+
+The `azure_maps_travel_time` sensor provides travel time from the [Azure Maps Route Directions API](https://docs.microsoft.com/en-us/rest/api/maps/route/getroutedirections).
+
+## {% linkable_title Setup %}
+
+You need to register for an API key by following the instructions [here](https://docs.microsoft.com/en-us/azure/azure-maps/how-to-manage-account-keys). There are two keys there, both will work.
+
+Azure Maps is a paid Azure service, [pricing details here](https://azure.microsoft.com/en-us/pricing/details/azure-maps/). But you do get 25000 free API calls per month, so if you do not exceed 33 calls per hour then you incur no costs, if you want to have many travel times visible, consider changing the [scan interval](/docs/configuration/platform_options/#scan-interval) to something longer than 5 minutes to stay within the free credit limit or update the sensors on-demand using an automation (see example below).
+
+## {% linkable_title Configuration %}
+
+To enable the sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example entry for configuration.yaml
+sensor:
+  - platform: azure_maps_travel_time
+    api_key: XXXX_XXXXX_XXXXX
+    name: Home to Work
+    origin: zone.home
+    destination: zone.work
+```
+
+{% configuration %}
+api_key:
+  description: Your application's API key (get one by following the instructions above). This key identifies your application for purposes of quota management.
+  required: true
+  type: string
+origin:
+  description: "The starting point for calculating travel distance and time. You can supply a location in the form of a zone, device tracker or latitude/longitude coordinates
+  required: true
+  type: string
+destination:
+  description: One or more locations to use as the finishing point for calculating travel distance and time. The options for the destinations parameter are the same as for the origins parameter, described above.
+  required: true
+  type: string
+name:
+  description: A name to display on the sensor. The default is "Azure Maps Travel Time - [Travel Mode]" where [Travel Mode] is the mode set in options for the sensor (see option "travelMode" below).
+  required: false
+  type: string
+options:
+  description: "A dictionary containing parameters to add to all requests to the Route Directions API. A full listing of available options can be found [here](https://docs.microsoft.com/en-us/rest/api/maps/route/getroutedirections#uri-parameters)."
+  required: false
+  type: list
+  keys:
+    travelMode:
+      description: The travel mode used to calculate the directions/time. Can be `car`, `pedestrian` or `bicycle`.
+      required: false
+      default: car
+      type: string
+    departAt:
+      description: Can be iso timestamp, or a 24 hour time string like `08:00`. If you provide a time string, it will be combined with the current date to get travel time for that moment or the next day.
```

Comment:
```suggestion
      description: "Can be iso timestamp, or a 24 hour time string like `08:00`. If you provide a time string, it will be combined with the current date to get travel time for that moment or the next day."
```

Reply:
Thanks @klaasnicolaas 👍 
I could not merge in your suggest, but nevertheless, fixed it.